src/Mod/Part/parttests/regression_tests.py

```diff
@@ -20,6 +20,81 @@ def test_issue_4456(self):
         with self.assertRaises(IndexError):
             result.pop()
 
+    def test_issue_15735(self):
+        """
+        15735: Point in sketch as loft profile won't work in dev, but works in stable
+        The following test is a simplified version of the issue, but the outcome is the same
+        """
+
+        # Arrange
+        from FreeCAD import Base
+        import Sketcher
```

Comment:
Push this to top of file and quiet the linter?

Reply:
Absolutely.

I haven't enough experience in python tests, therefor I wasn't sure about what's the best way to handle this kind of warnings.

Thanks for the hint!